### ISSUE TITLE
`OSPSuite.R.Services.DataImporterTask` should check importer configuration.

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/OSPSuite.Core#1666]
Issue described in https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/2467

**Comment by PavelBal** [comment]
@rwmcintosh is PK-Sim updated already? I updated the libraries in this package and still getting the same error.

**Comment by rwmcintosh** [comment]
no, compatibility in OSPSuite.Core is broken so we can't update apps until everything works together again.
